src/CryptoInfo.h

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
this new file does not much sense
because include the CryptoMode
and both files have a single struct
its better merge CryptoInfo.h with CryptoMode.h in a single file something like CryptoUtils